### PR TITLE
update repositry details in `@opennextjs/aws` package.json

### PR DESCRIPTION
**Pull request by dario-piotrowicz**

### CODE REVIEW

**Comment by dario-piotrowicz** [comment]
@conico974 sorry my bad! now it should be all good 😓

**Comment by dario-piotrowicz** [comment]
> I think we need to wait for the next release for it to work well

ahhhh! ok good find! 👍 

yeah I guess it's probably that

**Review by conico974 — APPROVED** [review]
That's fine, we'll wait for the next release.
In the meantime we just have to look into stackblitz to get the correct url.

Thanks for that

File: packages/open-next/package.json
Comment (conico974) [inline packages/open-next/package.json:64]: @dario-piotrowicz Wait is this correct actually ? 
The one in cloudflare is just this `"url": "https://github.com/opennextjs/opennextjs-cloudflare.git"`
Maybe that's why it doesn't work

Comment (dario-piotrowicz) [reply packages/open-next/package.json:64]: mh... `git+` should be correct as that what's included in the npm docs: https://docs.npmjs.com/cli/v10/configuring-npm/package-json#repository
🤔 

I'm happy to give the http version a go too, I think that they're likely both valid

Comment (dario-piotrowicz) [reply packages/open-next/package.json]: @conico974 yeah the longer version is not helping, I do think we just need to wait for the next release 😕

Comment (conico974) [reply packages/open-next/package.json]: It was worth a try.
Ok i'll merge this then, we can revisit this if it still don't work on next release

**Pull request merged**
